### ISSUE TITLE
Can scroll content beneath the sidenav when the sidenav is opened.

### ISSUE THREAD

**Issue by yoav-zibin** [angular/material#8369]
**Actual behavior**:
-  What is the issue? *

Only on Chrome on iOS (doesn't happen on android):
Can scroll content when leftnav is opened.

-  What is the expected behavior?

When leftnav is opened, one shouldn't be able to scroll the content beneath it.

**CodePen** or Steps to reproduce the issue: *
-  CodePen Demo which shows your issue :

Take your side-nav demo:
https://material.angularjs.org/latest/demo/sidenav
but change the height from 
  style="height:500px;"
to
  style="height:150px;"
(so there will be a scrollbar on the content)

It's this code-pen:
http://codepen.io/anon/pen/NNoqPW 
but that code-pen doesn't work well on mobile ios, so I copied the HTML&JS here:
http://yoav-zibin.github.io/emulator/examples/scroll_leftnav_on_ios_bug.html

See video of the bug:
https://youtu.be/gJ72NkJxr0U
(I can scroll the content while the leftnav is opened).

**Angular Versions**: *
-  Angular Version: 1.5.5
-  Angular Material Version: rc4

**Additional Information**:
-  Browser Type: Chrome
-  Browser Version: *
-  OS: iOS
-  Stack Traces:

----
Shortcut to create a [new CodePen Demo](http://codepen.io/team/AngularMaterial/pen/bEGJdd).
Note: `*` indicates required information. Without this information, your issue may be auto-closed.

> Do not modify the titles or questions. Simply add your responses to the ends of the questions.
  Add more lines if needed.

**Comment by yoav-zibin** [comment]
Man, I just realized that the material site:
https://material.angularjs.org/latest/
is broken in exactly the same way on chrome ios:
you can scroll the content (when scrolling on the backdrop of the leftnav) while the leftnav is open :)

**Comment by graphefruit** [comment]
Have a look here for a fast fix: https://github.com/angular/material/issues/1637#issuecomment-76347652

This works on my project aswell.

**Comment by yoav-zibin** [comment]
Thank you so much for the pointer! I love this community :)

Your pointer helped me create a workaround, so I'm closing this issue (it's kinda of a duplicate of #1637 anyway).

I had some failed attempts along the way, so I'll document it here for future me and others :)

My md-content is the one that had :
  overflow-y: scroll;
and it's not a parent of the SideNav, so the fix here didn't help my situation:
https://github.com/angular/material/commit/dc2d204d9ac6c3536fbc485ff555cc419a601391

So I tried moving the "overflow-y: scroll" to the parent of SideNav, which created really weird problems on ios (where if you scroll fast, the entire page becomes white).

So then I tried the solution where you track whether SideNav is open or not, and change overflow-y accordingly, which worked with some tweaking because my UI is responsive (and when SideNav is locked open, sideNav.isOpen returns the old open/close attribute).
So my final working solution is:

CSS:

.overflow-y-scroll {
  -webkit-overflow-scrolling: touch;
  overflow-y: scroll;
}
.not-scrollable {
  overflow-y: hidden;
}


JS:
$rootScope.sideNavIsOpen = () => false; // overridden later.
      $mdComponentRegistry.when('left').then(function(sideNav: any) {
        $rootScope.sideNavIsOpen = () => sideNav.isOpen() &&
            // When gt-xs, then the sideNav is always open (and sideNav.isOpen may return true/false regardless).
            !$mdMedia('gt-xs');
      });



HTML:
    &lt;md-content flex ng-class="sideNavIsOpen() ? 'not-scrollable' : 'overflow-y-scroll'"&gt;
